### DISCUSSION TITLE
使用 Meta 内核导致英雄联盟超高延迟和多个出口问题

### DISCUSSION THREAD

**Discussion by zzz6839** [MetaCubeX/mihomo#57] [Q&A] [answered]
meta内核启用之后导致英雄联盟ping值不稳定和超高延迟。。。怎么办？

openclash整合，未开TCP并发和流量（域名）探测

用的alpha-g7136d14版本

**Comment by zzz6839** [comment, marked as answer]
emm,应该是自编译固件的玄学问题，更换固件解决，开了meta core ping值还下降了，好评

**Reply by DEAN-Cherry** [reply]
是 openWrt 的问题吗，我现在也出现这个问题。。。
